devops/geoapi-services/nginx.conf

```diff
@@ -38,15 +38,15 @@ http {
       large_client_header_buffers 2 50k;
 
       location / {
-           add_header "Access-Control-Allow-Origin"  *;
+          add_header 'Access-Control-Allow-Origin' '*' always;
 
           # Preflighted requests
-          if ($request_method = OPTIONS ) {
-              add_header "Access-Control-Allow-Origin"  *;
-              add_header "Access-Control-Allow-Methods" "GET, POST, OPTIONS, HEAD, PUT, DELETE";
-              add_header "Access-Control-Allow-Headers" "*";
-              add_header 'Access-Control-Max-Age' 1728000;
-              add_header 'Content-Length' 0;
+          if ($request_method = 'OPTIONS') {
+              add_header 'Access-Control-Allow-Origin' '*' always;
+              add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS, HEAD, PUT, DELETE' always;
+              add_header 'Access-Control-Allow-Headers' '*' always;
+              add_header 'Access-Control-Max-Age' 1728000 always;
```

Comment:
Does this actually override the browser limits enforced in FF, Chrome, etc?

`Maximum number of seconds the results can be cached, as an unsigned non-negative integer. Firefox [caps this at 24 hours](https://searchfox.org/mozilla-central/source/netwerk/protocol/http/nsCORSListenerProxy.cpp#1207) (86400 seconds). Chromium (prior to v76) [caps at 10 minutes](https://source.chromium.org/chromium/chromium/src/+/main:services/network/public/cpp/cors/preflight_result.cc;drc=52002151773d8cd9ffc5f557cd7cc880fddcae3e;l=36) (600 seconds). Chromium (starting in v76) [caps at 2 hours](https://source.chromium.org/chromium/chromium/src/+/main:services/network/public/cpp/cors/preflight_result.cc;drc=49e7c0b4886cac1f3d09dc046bd528c9c811a0fa;l=31) (7200 seconds). The default value is 5 seconds.`

Reply:
I could just remove the `Access-Control-Max-Age` line and go with the default.  thoughts?

Reply:
moving to 86400 on all these.